apps/aecore/src/aec_peers.erl

```diff
@@ -194,6 +196,18 @@ get_local_peer_uri() ->
 update_last_seen(Uri) ->
     gen_server:cast(?MODULE, {update_last_seen, uri(Uri), timestamp()}).
 
+%%------------------------------------------------------------------------------
+%% Check user-provided environment
+%%------------------------------------------------------------------------------
+check_env() ->
+    case aeu_env:user_config(<<"peers">>) of
+        {ok, Peers0} when is_list(Peers0) ->
+            Peers = [binary_to_list(P) || P <- Peers0],
+            application:set_env(aecore, peers, Peers);
```

Comment:
I see this overwrites values in sys.config. Needs mention in release notes.

Reply:
Yes, that is basically the point. I agree that it needs to be documented.

Reply:
So this code is reading from user config (e.g. `~/.epoch/NODENAME`) and overwriting with it (in memory) whatever was in `application:get_env` for that key (e.g. from `sys.config`) so that rest of the code in the app can use only plain `application:get_env` for querying.

We shall standardize on how to use these capabilities consistently across the apps - and this is a good start! No need to change this PR.

Reply:
1. we need to definitely let devops know
2. that's usual approach for consumer software, so I totally support it
3. with version upgrades we will need to introduce some sort of validation, to discard unsupported config from local config